dataactcore/config_example.yml

```diff
@@ -6,24 +6,34 @@
 broker:
 
     # Specify the url where the front end of the application will be accessed.
-    # For a local installation this will most likely be localhost or the 
-    # location where the /public files are located. If a port is required, 
+    # For a local installation this will most likely be localhost or the
+    # location where the /public files are located. If a port is required,
     # it should be appended to the URL.
-    full_url: http://dataactbroker.domain.com
+    full_url: http://localhost:3000
+    # full_url: https://alpha-broker-sandbox.usaspending.gov/
```

Comment:
This doesn't have to be here, this is from my local config file because I don't feel like remembering it all the time.